stubs/Flask-Migrate/flask_migrate/__init__.pyi

```diff
@@ -1,28 +1,53 @@
+# pyright: reportInvalidStubStatement=none
+
+import sys
+from _typeshed import StrPath, SupportsFlush, SupportsKeysAndGetItem, SupportsWrite
+from argparse import Namespace
 from collections.abc import Callable, Iterable, Sequence
 from logging import Logger
-from typing import Any, TypeVar
+from typing import Any, Protocol, TypeVar
 from typing_extensions import ParamSpec, TypeAlias
 
 import flask
 from flask_sqlalchemy import SQLAlchemy
 
 _T = TypeVar("_T")
+_T_contra = TypeVar("_T_contra", contravariant=True)
 _P = ParamSpec("_P")
 _ConfigureCallback: TypeAlias = Callable[[Config], Config]
+_AlembicConfigValue: TypeAlias = Any
 
 alembic_version: tuple[int, int, int]
 log: Logger
 
+class _SupportsWriteAndFlush(SupportsWrite[_T_contra], SupportsFlush, Protocol): ...
```

Comment:
`SupportsFlush` is new and not yet distributed with at least mypy. It's safer to do this for now:

```suggestion
# TODO: Use _typeshed.SupportsFlush when it's available in type checkers.
class _SupportsWriteAndFlush(SupportsWrite[_T_contra], Protocol):
    def flush(self) -> object: ...
```